pkg/odo/cli/pipelines/ui/ui.go

```diff
@@ -0,0 +1,246 @@
+	}
+	err := survey.AskOne(prompt, &path, survey.Required)
+	ui.HandleError(err)
+	return path
+}
+
+func EnterPrefix() string {
+	var path string
+	prompt := &survey.Input{
+		Message: "Add a prefix to the environment names(dev, stage,cicd etc.) to distinguish and identify individual environments?",
+		Help:    "The prefix helps differentiate between the different namespaces on the cluster, the default namespace cicd will appear as test-cicd if the prefix passed is test.",
+	}
+	err := survey.AskOne(prompt, &path, nil)
+	ui.HandleError(err)
+	return path
+}
+
+func EnterServiceRepoURL() string {
+	var path string
+	prompt := &survey.Input{
+		Message: "Provide the URL for your Service repository e.g. https://github.com/organisation/service.git",
+		Help:    "The repository name where the source code of your service is situated",
+	}
+	err := survey.AskOne(prompt, &path, survey.Required)
+	ui.HandleError(err)
+	return path
+}
+
+func EnterServiceWebhookSecret() string {
+	var path string
+	prompt := &survey.Input{
+		Message: " Provide a secret that we can use to authenticate incoming hooks from your Git hosting service for the Service repository. (if not provided, it will be auto-generated)",
+		Help:    "The webhook secret is a secure string you plan to use to authenticate pull/push requests to the version control system of your choice, this secure string will be added to the webhook sealed secret created to enhance security. Choose a secure string of your choice for this field.",
+	}
+	err := survey.AskOne(prompt, &path, nil)
+	ui.HandleError(err)
+	return path
+}
+
+func SelectOptionImageRepository() string {
+	var path string
+
+	prompt := &survey.Select{
+		Message: "Select type of image repository",
+		Options: []string{"Openshift Internal repository", "External Registry"},
+		Default: "Openshift Internal repository",
+	}
+	err := survey.AskOne(prompt, &path, survey.Required)
+	ui.HandleError(err)
+	return path
+}
+
+func SelectOptionOverwrite() string {
+	var path string
+
+	prompt := &survey.Select{
+		Message: "Do you want to overwrite your output path. Select yes or no",
+		Options: []string{"yes", "no"},
+		Default: "no",
+	}
+	err := survey.AskOne(prompt, &path, survey.Required)
+	ui.HandleError(err)
+	return path
+}
+
+func SelectOptionCommitStatusTracker() string {
+	var path string
+
+	prompt := &survey.Select{
+		Message: "Please enter (yes/no) if you desire to use commit-status-tracker",
+		Options: []string{"yes", "no"},
+	}
+	err := survey.AskOne(prompt, &path, survey.Required)
+	ui.HandleError(err)
+	return path
+}
+
+func SelectOptionBootstrap() string {
+	var path string
+	prompt := &survey.Select{
+		Message: "Please enter (Bootstrap/init), choose bootstrap if you wish to add a mock service to the gitops repository",
```

Comment:
```suggestion
		Message: "Please enter (Bootstrap/Initialise), choose bootstrap if you wish to add a mock service to the gitops repository",
```

Reply:
This no longer exists due to the fact that the bootstrap/init process takes place based on whether we pass in the service repo